AwsEncryptionSDK/runtimes/net/Examples/Keyring/AwsKmsHierarchical/AwsKmsHierarchicalKeyring.cs

```diff
@@ -34,9 +37,13 @@
 /// or you can implement an interface that selects 
 /// the Branch Key based on the Encryption Context.
 /// 
-/// This example demonstrates configuring a Hierarchical Keyring
-/// with a Branch Key ID Supplier to encrypt and decrypt data for
-/// two separate tenants.
+/// This example demonstrates how to:
+/// - Create a key store
+/// - Create a branch key
+/// - Version a branch key
+/// - Configure a Hierarchical Keyring
+///   with a static branch key configuration to ensure we are restricting
+///   access to a single tenant. 
 /// 
 /// This example requires access to the DDB Table where you are storing the Branch Keys.
```

Comment:
Suggestion: Specify what access you need.
Also, you SHOULD require DescribeTable permission.
```suggestion
/// This example requires read & write access to the DDB Table where you are storing the Branch Keys.
```